test/HazelcastClientTest.js

```diff
@@ -82,30 +57,31 @@ configParams.forEach(function (cfg) {
         before(async function () {
             cluster = await RC.createCluster(null, null);
             await RC.startMember(cluster.id);
-            cfg.clusterName = cluster.id;
-            client = await Client.newHazelcastClient(cfg);
+            client = await Client.newHazelcastClient({
```

Comment:
BTW, we are creating the client in beforeEach in 5.x series. It might also be better here for test isolation as getDistributedObjects interacts with the proxy manager

Reply:
Ok I will change all of them to be like that.